news/_posts/translation/2021-11-09-do-you-speak-prestashop-october-2021.md

```diff
@@ -0,0 +1,121 @@
+| 6. | M. Mirena (mmirena) | Albanian | 467
+| 7. | slstranslation | Korean | 467
+| 8. | Rauno Riikman (weaver) | Finnish | 359
+| 9. | Rúben Martins (RubenMartins) | Portuguese | 334
+| 10. | Store (HelaBasa) | Sinhala | 287
+ 
+#### Top 8 proofreaders in October:
+ 
+| | Proofreader | Language | # Words
+|-| ---------- | -------- | ----------------
+| 1. | Zoran Tejic (zotamal) | Serbian (Latin) | 1,970
+| 2. | 戢斌斌 (adonis_cn) | Chinese Simplified | 1,911
+| 3. | webdvl (megashopba) | Slovak | 908
+| 4. | Store (HelaBasa) | Sinhala | 564
+| 5. | Girdman.com (anargaragezov) | Azerbaijaini | 559
+| 6. | M. Mirena (mmirena) | Albanian | 467
+| 7. | crowd-dyr | Danish | 310
+| 8. | Arman Hosseini (Arman-Hosseini) | Persian | 29
+
+
+Congratulations, and welcome to our new top contributors! :clap:
+ 
+Remember, you can see who has been contributing to our translation project thanks to the [Translators page](https://translators.prestashop.com/).
+ 
+## Complete translations
+ 
+#### Fully available languages
+ 
+At the end of October, PrestaShop 1.7.8 was already fully available (= 100% translated and validated) in **14 languages**. That’s great! Thank you for your involvement. :tada:
+ 
+| Danish | Chinese traditional | English | Estonian |
+| French | Greek | Italian | Lithuanian |
+| Portuguese, Brazil | Romanian | **Serbian** | Spanish |
+| Spanish, Mexico | Swedish |
+
+Thank you all for your contribution! And remember, in order to get back to a 100% completion rate, all the [new 1.7.8 strings](https://build.prestashop.com/news/prestashop-178-translations/) should be translated and validated... Your project needs you! :muscle: 
+
+#### Almost there!
+
+Korean and Czech are almost fully available. There is less than 5% of strings to proofread. 
+There isn’t much work left, so if you wish to help to proofread what has been translated, please send an email to translation@prestashop.com or write me (Julie Varisellaz) on [Slack](https://join.slack.com/t/prestashop/shared_invite/zt-dkmbz5qf-I~FlEWwmRUOXunc5ui0Ucg). Your help is needed!
+
+## Languages with the best evolution
+
+The following languages had the best progress in October thanks to the translation community:
+ 
+* **Serbian** (+3% to reach 100% translated and validated) 
+* **Chinese Simplified** (+3% to reach 94% translated and validated)
+* **Persian** (+2% to reach 100% translated)
+
+Congratulations! 
+ 
+![Best translation progress in October 2021](/assets/images/2021/11/build-crowdin-progress-oct21.png)
+
+A huge thank you to all the contributors!
+ 
+Of course, this is highlighting the languages that made some progress with new translations; but it does not mean that the languages that are not mentioned here are not active. Indeed, some editing and rewriting could be going on, but the percentage of translation would not be modified (since the work happens on strings that are already translated). So let's not forget about the work of other contributors! Thanks to you too!
```

Comment:
```suggestion
Of course, we highlighted the languages that made some progress with new translations; but it does not mean that the languages that are not mentioned here are not active. Indeed, some editing and rewriting may be going on, but the percentage of translation would not be modified (since the work happens on strings that are already translated). So let's not forget about the work of other contributors! Thanks to you too!
```

Reply:
```suggestion
Of course, this is highlighting the languages that made some progress with new translations; but it does not mean that the languages that are not mentioned here are not active. Indeed, some editing and rewriting may be going on, but the percentage of translation would not be modified (since the work happens on strings that are already translated). So let's not forget about the work of other contributors! Thanks to you too!
```